src/state/slices/opportunitiesSlice/selectors/stakingSelectors.ts

```diff
@@ -689,7 +689,9 @@ export const selectUnderlyingStakingAssetsWithBalancesAndIcons = createSelector(
                 .div(bn(10).pow(asset?.precision ?? underlyingAsset?.precision ?? 1))
                 .toFixed(),
               icons: [underlyingAssetsIcons[i]],
-              allocationPercentage: bn('1').div(original.length).toString(),
+              allocationPercentage:
+                userStakingOpportunity.underlyingAssetRatios?.[i] ??
```

Comment:
This new props permis to display the real balancing percentages in each underlying pools/assets

Reply:
Same question as below

Reply:
Discussed this one out and actually LFG